test_files/es6/decorator.js

```diff
@@ -7,7 +7,11 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
 var __metadata = (this && this.__metadata) || function (k, v) {
     if (typeof Reflect === "object" && typeof Reflect.metadata === "function") return Reflect.metadata(k, v);
 };
-function decorator(/** Object */ a, /** string */ b) { }
+/**
+@param { Object} a
```

Comment:
usually these guys look like 

/** 
 * @param ...
 * @return 
 */
Notice the extra *. My guess is that closure doesn't care, but we should try to add them if it is not too hard.